Documentation/network/kubernetes/policy.rst

```diff
@@ -50,10 +50,6 @@ Known missing features for Kubernetes Network Policy:
 +===============================+===================+
 | ``ipBlock`` set with a pod IP | :gh-issue:`9209`  |
 +-------------------------------+-------------------+
-| SCTP                          | :gh-issue:`5719`  |
```

Comment:
If SCTP is now supported, we should probably consider adding SCTP to the upstream k8s suite in `test/kubernetes-test.sh`. It looks like it's skipped for now.

Reply:
Logically it'd be nice to process the SCTP change separately to ensure proper scrutiny. If it's really supported on older versions then this could also be a candidate for fixing the docs in older branches.